packages/admin-api-client/src/validations.ts

```diff
@@ -0,0 +1,23 @@
+import { CLIENT } from "./constants";
+
+export function validateRequiredStoreDomain(storeDomain: string | undefined) {
+  if (
+    !storeDomain ||
+    typeof storeDomain !== "string" ||
+    storeDomain.trim().length < 1
+  ) {
+    throw new Error(`${CLIENT}: a valid store domain must be provided`);
+  }
+}
+
+export function validateRequiredAccessToken(accessToken: string) {
+  if (!accessToken) {
+    throw new Error(`${CLIENT}: an access token must be provided`);
+  }
+}
+
+export function validateServerSideUsage() {
```

Comment:
This might be useful enough to move to the common validation utils in `graphql-client`

Reply:
@scottdixon - are you planning to move this over? We can do it in a follow up PR if that's the plan but just wanted to understand if there's next steps to this comment.